web/packages/teleport/src/Apps/MCPAppConnectDialog.tsx

```diff
@@ -78,14 +95,41 @@ export function MCPAppConnectDialog(props: { app: App; onClose: () => void }) {
               <Text bold as="span">
                 Step 2
               </Text>
-              {' - Log in the MCP server'}
+              {' - Configure your MCP client'}
             </Text>
-            <TextSelectCopy text={`tsh mcp login ${app.name}`} />
+            <Flex alignItems="center" justifyContent="left" columnGap={2}>
+              <Link href={links.cursor} target="_blank">
+                <ResourceIcon name="mcpCursor" height="32px" />
+              </Link>
+              <Link href={links.vscode} target="_blank">
+                <ResourceIcon name="mcpVscode" height="25px" />
+              </Link>
+              <Link href={links.vscodeInsiders} target="_blank">
+                <ResourceIcon name="mcpVscodeInsiders" height="25px" />
+              </Link>
+            </Flex>
+            <Box>
+              Here is a sample Claude Desktop config to connect to this MCP
+              server:
+            </Box>
+            <TextSelectCopyMulti
+              bash={false}
+              lines={[
+                {
+                  text: claudeConfig,
+                },
+              ]}
+            />
+            <Box>
+              Alternatively, run the following to generate the config from the
+              command line.
+            </Box>
+            <TextSelectCopy text={`tsh mcp config ${app.name}`} />
+            <Box>
+              *Note: You might need to restart your MCP client to load the
```

Comment:
The asterisk here feels a bit out of place because there's no corresponding asterisk within the main body of the text. I think we could drop the asterisk and "Note:" would work just as well.